dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/guestbook/GuestbookPage.java

```diff
@@ -3,21 +3,21 @@
  * To change this template file, choose Tools | Templates
  * and open the template in the editor.
  */
-package edu.harvard.iq.dataverse;
+package edu.harvard.iq.dataverse.guestbook;
```

Comment:
IMO it would be more suitable to move it to `edu.harvard.iq.dataverse.dataverse.guestbook` since guestbooks belongs to dataverse (Not sure if this is the right approach, but I think if we would divide some packages so they would belong under dataverse or dataset it would be better for navigation) (We should probably change the main package name in the future since it is kinda confusing).

Reply:
I don't want to make a change like that without consulting. If we decide to move packages then IMO we should do it in a separate task and with a single approach in mind. 
In case of guestbooks it's not that obvious where should they be placed since they are connected with both dataverse and dataset objects.

Reply:
I don't necessarily agree that moving single object needs a task since I agree with "boy scout rule", but I agree that we need a discussion how to approach packaging, and maybe have a task with moving everything.

Reply:
Yes, I meant a task for everything.